javascript/operators/await.json

```diff
@@ -70,11 +70,11 @@
             "spec_url": "https://tc39.es/proposal-top-level-await/#introduction",
             "support": {
               "chrome": {
-                "version_added": false,
+                "version_added": "89",
                 "notes": "See <a href='https://crbug.com/v8/9344'>bug 9344</a>"
```

Comment:
If we're not going to include the flag data, then we ought to remove the note too.

```suggestion
```